.circleci/config.yml

```diff
@@ -1123,11 +1133,14 @@ jobs:
                     --services "<< parameters.service >>" \
                     --hosts "<< parameters.host >>" \
                     << parameters.params >>
-      - unless:
-          condition: << parameters.host >>
+      - when:
+          condition:
+            and:
+              - not: << parameters.host >>
+              - not: << parameters.deploy_args >>
           steps:
             - run:
-                name: Perform Deployment
+                name: Perform Deployment (w/o --hosts)
```

Comment:
nit, i think we should just call this step Perform Deployment and drop (w/o hosts), i think that will lead to more confusion